docs/Create-a-simple-example-with-NPM.md

```diff
@@ -10,21 +10,21 @@ echo "{}" > package.json
 
 First, Install BuckleScript.
 (For more advacned settings, please read
-[Installation](./Installation))
+[Installation](./Installation.md))
 
 
 
 ```sh
-npm install bs-platform --save 
```

Comment:
Sorry, a bit of noise here introduced by my editor's setting of trimming dangling whitespace.